src/main/java/com/codejoust/main/model/User.java

```diff
@@ -37,8 +45,23 @@
      */
     private String sessionId;
 
+    // This column holds the primary key of the account
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "account_table_id")
+    private Account account;
```

Comment:
I left a few comments about the SO post and moving JoinColumn to the OneToMany side to get it to work, but then I just realized that this seems to work fine? There isn't an associated field in Account and there's no extra generated table in the database - lmk if I'm missing something

Reply:
So I realized both work just fine, but that the `OneToMany` associations need the `@Fetch(value = FetchMode.SUBSELECT)`annotation, whereas the `ManyToOne` association doesn't need that. That's why this works fine, but I thought that the `OneToMany` associations didn't work initially.